schedule/yam/agama/agama_install_with_cryptfs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: agama_installation
+description: >
+  Playwright test on agamaq
+vars:
+  ENCRYPT: 1
+  DESKTOP: textmode
+schedule:
+  - installation/bootloader_start
+  - yam/agama/patch_agama
+  - yam/agama/agama_install_with_cryptfs
+  - installation/boot_encrypt
+  - installation/first_boot
+  - console/validate_encrypt
+test_data:
```

Comment:
same for test data, better using YAML_TEST_DATA.

Reply:
Done.